app/Database/Tables.hs

```diff
@@ -224,23 +219,38 @@ instance FromJSON SvgJSON
 -- JSON encoding/decoding
 instance FromJSON Courses where
```

Comment:
I started reviewing this function and then realized that I don't think we need this anymore? Since we're now parsing course information just from the calendar, but not from the timetable API. If so we can delete this code.